ssh/src/main/java/de/eidottermihi/rpicheck/ssh/impl/RaspiQuery.java

```diff
@@ -75,6 +75,9 @@
     private static final String DISK_USAGE_CMD = "LC_ALL=C df -h";
     private static final String DF_COMMAND_HEADER_START = "Filesystem";
     private static final String DISTRIBUTION_CMD = "cat /etc/*-release | grep PRETTY_NAME";
+    private static final String KERNEL_CMD = "cat /proc/version|cut -f 3 -d \" \"";
+    private static final String ARCHITECTURE_CMD = "(uname -m;echo \" \"; getconf LONG_BIT; echo \"bit\")|tr -d '\\n'";
```

Comment:
I've checked with my last remaining Pi in use, which uses LibreElec: `getconf` is not avalailabe there:

```
LibreELEC:~ # (uname -m;echo " "; getconf LONG_BIT; echo "bit")|tr -d '\n'
armv7l -sh: getconf: not found
bit
```

Is there a proper way to only use `getconf` if its available?

Reply:
Good catch, I didn't realize this command is not posix comptible. I did come up with a solution:
`
(uname -m; [ -x "$(command -v getconf)" ] && echo " " && getconf LONG_BIT && echo "bit")|tr -d '\n'
`
I'll commit this if you are happy with it.

Reply:
Updated it with your solution in https://github.com/eidottermihi/rpicheck/commit/95f3510d2fb1ad3d7abcf0fbf11e883aa6a4d233, had some trouble to update the PR directly.

Thanks for your contribution, great stuff!